C4SGSvc/src/main/java/org/c4sg/constant/UserDisplay.java

```diff
@@ -1,9 +1,29 @@
 package org.c4sg.constant;
 
 public enum UserDisplay {
```

Comment:
I just have supported the old implementation. I see that it would be better to user Boolean